flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableConfig.java

```diff
@@ -57,6 +57,18 @@
 	 */
 	private Integer maxGeneratedCodeLength = 64000; // just an estimate
 
+	/**
+	 * Specifies the name of the initial catalog to be created when instantiating
+	 * TableEnvironment.
+	 */
+	private String bultinCatalogName = "default-catalog";
+
+	/**
+	 * Specifies the name of the default database in the initial catalog to be created when instantiating
+	 * TableEnvironment.
+	 */
+	private String bultinDatabaseName = "default-database";
```

Comment:
Having '-' in the names might  confuse the parser as it can be read as a numeric minus operation.